examples/example_sklearn.py

```diff
@@ -4,6 +4,18 @@
 import matplotlib.pyplot as plt
 import pandas as pd
 
+import sys
+import importlib.util
+
+is_available = importlib.util.find_spec("sklearn")
+if not is_available:
+    print(
+        """scikit-learn needs to be installed for this example.
+You can install it by running the following command:
```

Comment:
Please re-phrase, because people may also be using other package managers, most notably conda. 
Maybe: "If you are using pip as package manager, you can install it by running the following command:"

Reply:
Indeed, I did not think about other package managers when I wrote this warning.
I changed the line to “If you use pip as a package manager, you can install it with the following command:”.